src/mlagility/cli/cli.py

```diff
@@ -93,6 +94,14 @@ def main():
     # Parser for the "benchmark" command
     #######################################
 
+    def check_extension(choices, file_name):
+        _, extension = os.path.splitext(file_name.split("::")[0])
+        if extension[1:].lower() not in choices:
+            raise exceptions.ArgError(
+                f"input_files must end with .py or .onnx (got '{file_name}')"
```

Comment:
```suggestion
                f"Unexpected positional argument `benchit {sys.argv[1]}`. The positional argument must either be a valid command (see `benchit -h`) or an input file with the .py or .onnx file extension."
```

Would you kindly expand this to cover incorrect or misspelled commands as well? With the current code if I were to do `benchit cachr -h` the error message would be `input_files must end with .py or .onnx (got cachr)`.

With my suggestion, it would be like:

`benchit cachr -h`

```
Unexpected positional argument `benchit cachr`. The positional argument must either be a valid command (see `benchit -h`) or an input file with the .py or .onnx file extension.
```

Reply:
Done! Also checking for typos now.

Reply:
Thank you!